web/src/server_events.js

```diff
@@ -281,8 +281,8 @@ export function force_get_events() {
     get_events_timeout = setTimeout(get_events, 0);
 }
 
-export function home_view_loaded() {
-    waiting_on_homeview_load = false;
+export function fetched_latest_message() {
+    waiting_on_fetching_latest_message = false;
```

Comment:
I suppose `all_messages_data_contains_latest_message` would be the most precise name I can come up with for this variable.

Reply:
I dropped this commit actually since we decided to remove this behaviour in the last commit.